apis/kueue/v1beta1/clusterqueue_conversion.go

```diff
@@ -38,12 +38,56 @@ func (dst *ClusterQueue) ConvertFrom(srcRaw conversion.Hub) error {
 
 func Convert_v1beta1_ClusterQueueSpec_To_v1beta2_ClusterQueueSpec(in *ClusterQueueSpec, out *v1beta2.ClusterQueueSpec, s conversionapi.Scope) error {
```

Comment:
I know we have #7394 but maybe we could consider writing some unit tests for this conversion.

Reply:
Sure we need unit tests here, but I'm not too concerned about them in this PR. Every CQ goes via conversion as storage is still in v1beta1. I think we can follow up in next PR.